pgwidget/pgwidget_core.py

```diff
@@ -1420,9 +1421,12 @@ def on_unclick(self, glc):
         
 
 class Entry(TextContainerRect):
-    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],is_password=False,selection_color=c.red):
-        self.is_password=is_password #must be initialized before text definition (in super())
+    def __init__(self, text, pos, size, color=c.white, is_draggable=True, relative_pos=[0, 0],type='text',selection_color=c.red, file_types=None):
+        self.type=type #must be initialized before text definition (in super())
```

Comment:
did you check whether "is?password" is not used in the predecessor - according to the comment here?